src/flp/gadgets.rs

```diff
@@ -30,14 +30,20 @@ pub struct Mul<F: FieldElement> {
     num_calls: usize,
 }
 
+macro_rules! gadget_poly_fft_mem_len {
```

Comment:
Is a macro necessary here? I think a function marked `#[inline]` would be easier to read.

Reply:
Done.